app/services/open_library_service.rb

```diff
@@ -0,0 +1,31 @@
+class OpenLibraryService
+  include HTTParty
+  base_uri 'https://openlibrary.org'
+
+  def initialize(isbn)
+    @isbn = isbn
+  end
+
+  def book_info
+    info = send_request.parsed_response
+    formatted_response(info)
```

Comment:
I think you should use `info.body` here

Reply:
I get 'nil' because I'm not getting a 'body' element from `send_request.parsed_response`. This is info (`info = send_request.parsed_response`):

<img width="1399" alt="screen shot 2018-10-04 at 10 05 31 am" src="https://user-images.githubusercontent.com/12885969/46475952-4c70b000-c7bd-11e8-930e-cda4733b1dd3.png">